roles/kubernetes/node/defaults/main.yml

```diff
@@ -31,6 +31,11 @@ kubelet_cgroups_per_qos: true
 # Set to empty to avoid cgroup creation
 kubelet_enforce_node_allocatable: "\"\""
 
+# Set runtime cgroups
+kubelet_runtime_cgroups: "/systemd/system.slice"
```

Comment:
why runtime_cgroups use /systemd/system.slice?https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/node-allocatable.md#recommended-cgroups-setup

Reply:
Because minikube default is it. Is it wrong?

```
$ /c/apps/k8s/minikube-windows-amd64 ssh sudo systemctl status
...
[[0;1;32m●[[0m minikube
    State: [[0;1;32mrunning[[0m
     Jobs: 0 queued
   Failed: 0 units
    Since: Thu 2018-08-30 12:53:11 UTC; 2min 14s ago
   CGroup: /
           ├─init.scope
           │ └─1 /sbin/init noembed norestore
           └─system.slice
             ├─nfs-mountd.service
             │ └─2056 /usr/sbin/rpc.mountd
             ├─systemd-timesyncd.service
             │ └─1962 /usr/lib/systemd/systemd-timesyncd
             ├─crio.service
             │ └─2171 /usr/bin/crio --storage-driver=overlay2 --log-level=
```